test/e2e/run.sh

```diff
@@ -415,13 +419,15 @@ helm-launch() { # script API
         fi
     fi
 
-    timeout=$(timeout-for-deadline $deadline)
-    timeout=$timeout wait-config-node-status $cfgresource
+    if [[ -n "$cfgresource" ]]; then
```

Comment:
nit: does this really need [[. If not I'd go with [ for better local contextual consistency with the rest of the function.

Reply:
Fixed in #523.